swarm/storage/error.go

```diff
@@ -0,0 +1,13 @@
+package storage
+
+const (
+	ErrCustom = iota
```

Comment:
Would it not be simpler for these to just be individual errors?

For example:

```go
ErrAccess = errors.New("resource: access denied")
```

Reply:
Maybe easier, but I like the ability to add more context detail of each error type.